spec/models/computacenter/outgoing_api/cap_update_request_spec.rb

```diff
@@ -87,5 +87,20 @@
         expect { request.post! }.to raise_error(Computacenter::OutgoingAPI::Error)
       end
     end
+
+    context 'when the response contains an error' do
+      let(:response_body) do
+        '<CapAdjustmentResponse dateTime="2020-08-21T12:30:40Z" payloadID="11111111-1111-1111-1111-111111111111"><HeaderResult errorDetails="Non of the records are processed" piMessageID="11111111111111111111111111111111" status="Failed"/><FaledRecords><Record capAmount="9" capType="DfE_RemainThresholdQty|Std_Device" errorDetails="New cap must be greater than or equal to used quantity" shipTO="11111111" status="Failed"/></FaledRecords></CapAdjustmentResponse>'
```

Comment:
There's a typo here: `<FaledRecords` should be `<FailedRecords`

Reply:
This was copied so it's possible this is a genuine error from the api we're working with

Reply:
I checked with them over email - it was a typo in a previous version of their API, which has since been fixed. 

```ruby
[18] pry(main)> puts response.body.to_s
<CapAdjustmentResponse dateTime="2020-09-14T21:55:37Z" payloadID="bcd794dc-e73e-4527-9619-2d8070b77c75"><HeaderResult piMessageID="059df5cdf6d511eab0cc00000035c636" status="Success"/><FailedRecords/></CapAdjustmentResponse>
```

Reply:
Good work. I've fixed the typo. I've also added an extra test case with the success sample above to ensure that works as expected

Reply:
awesome! 👍